src/commands.rs

```diff
@@ -1274,7 +1274,11 @@ struct StatusArgs {}
 #[derive(clap::Args, Clone, Debug)]
 struct LogArgs {
     /// Which revisions to show
-    #[clap(long, short, default_value = ":heads()")]
+    #[clap(
+        long,
+        short,
+        default_value = "remote_branches().. | (remote_branches()..)-"
```

Comment:
Will the currently-checked-out commit always be shown?

Reply:
Yep, no difference from before anyway. The `remote_branches()..` is short for `remote_branches()..heads()`, which should include the current checkout.

You can actually end up in a state where the checkout is not a visible head (e.g. if you undo an operation that moved the checkout without undoing an operation that made the commit hidden). That's something I plan to add some warning about (something like "Your working copy is hidden. Use `jj co <some commit>` to check out a visible commit, or <some command> to unhide it.").